### PR TITLE
enable file-headers option of llvm-objdump for XCOFF object files

### PR DESCRIPTION
**Pull request by diggerlin**
the patch enable file-headers option of llvm-objdump for XCOFF object files

### CODE REVIEW

File: llvm/test/tools/llvm-objdump/XCOFF/file-headers.test
Comment (jh7370) [inline llvm/test/tools/llvm-objdump/XCOFF/file-headers.test:1]: Nit: newer tools tests use `##` for comments, to distinguish from lit directives like `RUN:`

File: llvm/test/tools/llvm-objdump/XCOFF/file-headers.test
Comment (jh7370) [inline llvm/test/tools/llvm-objdump/XCOFF/file-headers.test:5]: I suggest adding `--match-full-lines` to both `FileCheck` calls. Otherwise, there could be trailing output on the line that won't be picked up (for example, the 32-bit case will match both `start address: 0x00001111` and `start address: 0x0000111122223333` without it).

File: llvm/test/tools/llvm-objdump/XCOFF/file-headers.test
Comment (jh7370) [inline llvm/test/tools/llvm-objdump/XCOFF/file-headers.test:21]: What impact do these actually have on the output?

Comment (diggerlin) [reply llvm/test/tools/llvm-objdump/XCOFF/file-headers.test:21]: thanks , I delete this one.

File: llvm/test/tools/llvm-objdump/XCOFF/file-headers.test
Comment (diggerlin) [reply llvm/test/tools/llvm-objdump/XCOFF/file-headers.test:5]: thanks

**Review by chenzheng1030 — APPROVED** [review]
LGTM. Please wait for @jh7370 's approval. Thanks very much.

**Review by jh7370 — APPROVED** [review]
LGTM!

**Pull request merged**
